Dockerfile-users-irods

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs-irods:4.2.6-buster
+FROM hydroshare/hs-irods:2e7929c
```

Comment:
Any reason the tag doesn't follow the name convention?

Reply:
it's because we have an action that builds every commit and publishes to dockerhub
...
think we should add more descriptive tags? :)

Reply:
That is reasonable.